cpu/cc26x0/ccfg.c

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2016 Leon George
+ * (who won't go after you should you copy any part of this file ;-)
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+/**
+ * @ingroup     cpu_cc26x0
+ * @{
+ *
+ * @file
+ * @brief       CCFG-definition
+ *
+ */
+
+/* TODO this code is not used. someone would have to add a section to the linker script */
```

Comment:
so if it is not used, either remove this or have a look at the kinetis linkerscript (`CPU/kinetis_common/ldscripts/kinetis.ld`). The handling of the `fcfield` should be quite similar to the `ccfg` stuff...